conf/illumina.config

```diff
@@ -1,6 +1,7 @@
 // Illumina specific params
 
 params {
+
     // Repo to download your primer scheme from
     schemeRepoURL = 'https://github.com/artic-network/primer-schemes.git'
```

Comment:
Can the `jd2112` primer repo be used here? If you have to add more to the `jd2112` repo, then perhaps do not bother, but if the information is in the `jd2112` repo and all you have to do is change the address, then please do so